json_schemas/briefs-digital-outcomes-and-specialists-digital-specialists.json

```diff
@@ -188,6 +194,7 @@
     "numberOfSuppliers",
     "organisation",
     "priceWeighting",
+    "requirementsLength",
```

Comment:
How would I go about doing a migration? And does it just mean giving all published briefs a requirementsLength of 2 weeks?